scripts/bintray-add-package.py

```diff
@@ -103,7 +103,7 @@ def get_value(self, string, key):
         for char in "\"'\n":
             value = value.replace(char, '')
 
-        return value
+        return value.split(",")
```

Comment:
This will break fields like "description". Split should be done after extracting value.